test/gurka/test_gtfs.cc

```diff
@@ -323,6 +324,13 @@ TEST(GtfsExample, MakeProto) {
       std::string fname = transit_file_itr->path().string();
       mjolnir::Transit transit = mjolnir::read_pbf(fname);
 
+      if (std::isdigit(fname.back())) {
+        // we produce 2 pbf tiles on purpose, where the last one (xx.pbf.0) only has a bunch of stop
+        // pairs
+        EXPECT_NE(transit.stop_pairs_size(), 0);
+        continue;
+      }
```

Comment:
with `mjolnir.transit_pbf_limit: 1` we get two PBFs (one with `xx.pbf.0`, but the last one only has a few stop pairs, most of the objects are in the first one